config/environments/development.rb

```diff
@@ -103,4 +103,6 @@
   config.x.gitis.service_url = ENV.fetch('CRM_SERVICE_URL', 'notset')
   config.x.gitis.channel_creation = ENV.fetch('CRM_CHANNEL_CREATION', '0')
   config.x.gitis.country_id = ENV.fetch('CRM_COUNTRY_ID', SecureRandom.uuid)
+
+  config.ab_threshold = Integer ENV.fetch('AB_TEST_THRESHOLD', 100)
```

Comment:
The AB threshold is specific to this usage so should probably reference that in the naming?